src/Provider/SecurityServiceProvider.php

```diff
@@ -57,8 +59,6 @@ function ($app) {
                 ];
             }
         );
-
-        $app->register(new SilexSecurityServiceProvider());
     }
```

Comment:
You know the `$app['security.access_rules']` wasn't getting applied right? because the SP overrides that key when it's registered after you declare it.